client/src/Components/Pages/Search/Header/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+
+import AutosuggestComponent from "../AutoSuggest";
+
+import { HeaderWrapper, SearchBarContainer } from "./Header.style";
+import Tabs from "./Tabs";
+import Heading from "./Heading";
+
+const orgs = {
+  agency: { text: "Agencies", to: "agency" },
+  payroll: { text: "Payrolls", to: "payroll" },
+  worksite: { text: "Worksites", to: "worksite" },
+  company: { text: "Companies", to: "company" },
+};
+
+export default class SearchHeader extends Component {
+  state = {
+    shrink: false,
+  };
+
+  componentDidMount() {
+    document.addEventListener("scroll", this.checkScroll);
+    this.checkScroll();
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("scroll", this.checkScroll);
+  }
+
+  checkScroll = () => {
+    if (
+      document.body.scrollTop > 60 ||
+      document.documentElement.scrollTop > 60
+    ) {
+      this.setState({ shrink: true });
+    } else {
+      this.setState({ shrink: false });
+    }
+  };
```

Comment:
This is great work on the scroll shrinking the top header :)

Reply:
yeah there are some (forget password, report, privacy & terms, review, profile, reply)
I think it would be better to work on them on a seperated issue, what do you think?